README.md

```diff
@@ -521,6 +527,25 @@ module.exports = {
 };
 ```
 
+#### isTestRunner
+
+The `isTestRunner` function can be used to determine if a story is rendering in the context of the test runner.
+
+```js
+import { isTestRunner } from '@storybook/test-runner/is-test-runner';
+export const MyStory = () => (
+  <div>
+    <p>Is this story running in the test runner?</p>
+    <p>{isTestRunner() ? 'Yes' : 'No'}</p>
+  </div>
+);
+```
+
+The result of `isTestRunner()` will be true in the following scenarios:
+
+1. In the browser, when the story is rendered while running the test runner
+2. In node, if you prepend your Storybook script with `STORYBOOK_TEST_RUNNER=true`
```

Comment:
I'm sort of confused by this usage -- when would someone be using `isTestRunner` in node, and would they have to set the env var themselves?

Reply:
If people use the function inside of the play or render functions, or preview.js, it's all good, but if they use it in main.js or to setup a story parameter, the check would not work.

This would be the first step to make this check consistent - now involving setting up the var yourself, but once the test runner is able to spawn storybook (and forward environment variables), then users would not need to do anything and it would be always consistent

Reply:
thought about it more and just removed that bit entirely, it just adds confusion. Once the test-runner is able to spawn Storybook, then I'll bring it back